app/.server/content.ts

```diff
@@ -0,0 +1,69 @@
+import fs from 'node:fs/promises';
+import { glob } from 'glob';
+import { parseMarkdown } from './utils/parse-markdown';
+
+type Content = Awaited<ReturnType<typeof parseMarkdown>>;
```

Comment:
Is Awaited a built in typescript type to describe an "awaited" value? Neat. Not sure why it's needed though really

Reply:
If we lose the awaited, the type becomes wrapped in `Promise<>`, feels less "clean".